### PR TITLE
feat: remove zmq since edgex 2.3 not supporting it

### PR DESCRIPTION
**Pull request by vli11**
Signed-off-by: Valina Li <valina.li@intel.com>

### CODE REVIEW

**Review by brian-intel — APPROVED** [review]

**Review by jim-wang-intel — APPROVED** [review]

File: rtsf-at-checkout-device-scale/Dockerfile [hunk 1]
```diff
@@ -19,16 +19,14 @@ RUN go mod download
 COPY . .
 
 RUN go mod tidy
-RUN CGO_ENABLED=1 go build -o device-scale cmd/main.go
+RUN go build -tags no_zmq -o device-scale cmd/main.go
```
Comment (lenny-intel) [inline rtsf-at-checkout-device-scale/Dockerfile:22]: Note: In Minnesota (current Edgex WIP) this build tag has been removed with the removal of ZMQ.  So for the Minnesota EdgeX services/sdks this is not needed.

File: rtsf-at-checkout-loss-detector/Dockerfile [hunk 2]
```diff
@@ -7,7 +7,7 @@ LABEL license='SPDX-License-Identifier: Apache-2.0' \
     copyright='Copyright (c) 2019 - 2022: Intel'
 
 # add git for go modules
-RUN apk update && apk add --no-cache make git gcc libc-dev zeromq-dev linux-headers
+RUN apk update && apk add --no-cache make git linux-headers
```
Comment (lenny-intel) [inline rtsf-at-checkout-loss-detector/Dockerfile:10]: I think `linux-headers` can also be removed since that is a CGO thing.

**Review by lenny-intel — CHANGES_REQUESTED** [review]
One small change.

**Review by lenny-intel — APPROVED** [review]
LGTM

**Review by marcpfuller — APPROVED** [review]
LGTM

**Pull request merged**